telegram/ext/filters.py

```diff
@@ -2190,6 +2191,15 @@ def filter(self, message: Message) -> bool:
     )
     """Messages that contain :attr:`telegram.Message.proximity_alert_triggered`."""
 
+    class _RefundedPayment(MessageFilter):
```

Comment:
oh damn this reminds me we need a filter for PaidMedia, let me do that in another PR